src/components/Player/index.tsx

```diff
@@ -80,6 +101,30 @@ const PreviewContainer = styled.div`
   }
 `
 
+const Model3dContainer: React.FC<{ renderer?: string }> = ({ renderer }) => {
+  const [isLoading, setLoading] = useState(true)
+  useEffect(() => {
+    setLoading(true)
+  }, [renderer])
```

Comment:
`setLoading` is missing and `renderer` is not used in the hook

Reply:
After `renderer` changes, `onLoad` will be triggered again